batch/batch/driver/templates/index.html

```diff
@@ -37,6 +39,34 @@ <h1>Pools</h1>
     </tbody>
   </table>
 
+  <h2>Job Private</h2>
```

Comment:
Pools is a plural noun so I think we should call this `Job Private Instances`

Reply:
That doesn't make sense. It would have to be Job Private Instance Collection.